loom/common/assets/wavloader.cpp

```diff
@@ -0,0 +1,119 @@
+#include <string.h>
```

Comment:
After a relativley qucik search, I could not find code similar on the internet. It was probably written "from scratch" to use Loom support functions.